pkg/apis/triggers/v1alpha1/cluster_trigger_binding_types.go

```diff
@@ -43,7 +43,7 @@ type ClusterTriggerBinding struct {
 	Spec TriggerBindingSpec `json:"spec,omitempty"`
 
 	// +optional
-	Status TriggerBindingStatus `json:"status"`
+	Status TriggerBindingStatus `json:"status,omitempty"`
```

Comment:
Yeah, I don't think we're using status... nor do I think it makes sense for the TriggerBindings or TriggerTemplates to have a status 👍

Reply:
beware of removing it now though :angel: it might break users (see https://github.com/tektoncd/pipeline/blob/master/pkg/apis/pipeline/v1alpha1/pipeline_types.go#L72-L76 – we had to keep it for… not breaking user :stuck_out_tongue_closed_eyes: )

Reply:
Ahh good catch...though the 0.4 release is already a breaking change due to the API groups changing...so this might be the time to do it if we ever do this!

Reply:
@dibyom  So are we keeping the status field for bindings and templates? If not I can push the patch to remove status in the same PR

Reply:
Actually let's keep them with the `omitempty` for now. I think `omitempty` should fix @bigkevmcd 's issues with ArgoCD as well